projects/packages/plugin-deactivation/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+	"extends": "jetpack-js-tools/tsconfig.base.json",
+	"include": [ "src/assets/js/**/*" ],
```

I think you should be good with just
```suggestion
	"include": [ "./src/assets/js" ],
```

---

Done in 7cb96f15b1.